### PR TITLE
added new study BPDC000118 to dashboard publication page

### PR DESCRIPTION
**Pull request by cgmeyer**
Link to Jira ticket if there is one:

### CODE REVIEW

File: data.bloodpac.org/dashboard/Public/open-links/BPDC000118/index.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+  <table class="dcf-table dcf-table-responsive dcf-table-bordered dcf-table-striped dcf-w-100%">
+	<tbody>
+		<tr>
+			<td>Authors</td>
+			<td><span>Liya Xu<sup>1,2,†</sup>, Mary E. Kim<sup>1,3,†</sup>, Ashley Polski<sup>1,3</sup>, Rishvanth K. Prabakar<sup>4</sup>, Lishuang Shen<sup>5</sup>, Chen-Ching Peng<sup>1,2</sup>, Mark W. Reid<sup>1</sup>, Patricia Chévez-Barrios<sup>6</sup>, Jonathan W. Kim<sup>1,3</sup>, Rachana Shah<sup>7</sup>, Rima Jubran<sup>7</sup>, Peter Kuhn<sup>2,8,9,10</sup>, David Cobrinik<sup>1,3,8,11,12</sup>, Jaclyn A. Biegel<sup>5,13</sup>, Xiaowu Gai<sup>5,13</sup>, James Hicks<sup>2,8,11<\sup>, and Jesse L. Berry<sup>1,3,8,12,*<\sup>
```
Comment (mfshao) [inline data.bloodpac.org/dashboard/Public/open-links/BPDC000118/index.html:43]:
```suggestion
			<td><span>Liya Xu<sup>1,2,†</sup>, Mary E. Kim<sup>1,3,†</sup>, Ashley Polski<sup>1,3</sup>, Rishvanth K. Prabakar<sup>4</sup>, Lishuang Shen<sup>5</sup>, Chen-Ching Peng<sup>1,2</sup>, Mark W. Reid<sup>1</sup>, Patricia Chévez-Barrios<sup>6</sup>, Jonathan W. Kim<sup>1,3</sup>, Rachana Shah<sup>7</sup>, Rima Jubran<sup>7</sup>, Peter Kuhn<sup>2,8,9,10</sup>, David Cobrinik<sup>1,3,8,11,12</sup>, Jaclyn A. Biegel<sup>5,13</sup>, Xiaowu Gai<sup>5,13</sup>, James Hicks<sup>2,8,11</sup>, and Jesse L. Berry<sup>1,3,8,12,*</sup>
```

File: data.bloodpac.org/dashboard/Public/open-links/BPDC000118/index.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            <br>
+            L. Xu and M. Kim contributed equally as co-first authors of this article.
+            *corresponding author
+        </tr>
+		<tr>
+			<td>Affiliations</td>
```
Comment (mfshao) [inline data.bloodpac.org/dashboard/Public/open-links/BPDC000118/index.html:49]: kind of weird to have superscript text in authors but no `Affiliations`?

Comment (cgmeyer) [reply data.bloodpac.org/dashboard/Public/open-links/BPDC000118/index.html:49]: i will add those later, we just need to get the page up with the correct ID right now

**Review by mfshao — APPROVED** [review]
lgtm

**Pull request merged**